setup.py

```diff
@@ -13,9 +13,10 @@
 
 # get required packages from requirements.txt
 with open('requirements.txt', 'r') as fid_reqs:
-    # ignores commented lines and blank lines
+    # ignores commented lines, blank lines, and pytest
     reqs = [r.strip() for r in fid_reqs.read().splitlines()
-            if r and not r.strip().startswith('#')]
+            if r and not r.strip().startswith('#')
+            and 'pytest' not in r]
```

Comment:
I think now that we have the setup.py, we can drop the requirements.txt. Recent versions of the setup.py allow the same version identifiers and also other sections for pytest, etc. I'll show an example on Monday.

Reply:
I'm going to hold off on making any changes until after our meeting on Monday. In the meantime, should I delete this commit?